packages/flutter/lib/src/material/divider.dart

```diff
@@ -208,7 +208,7 @@ class VerticalDivider extends StatelessWidget {
       width: width,
```

Comment:
While you're here, remove the extra createBorderSide static and just use the other static